common/remote_object.go

```diff
@@ -246,6 +246,25 @@ func parseConsoleRemoteObjectPreview(logger *log.Logger, op *cdpruntime.ObjectPr
 	return string(bb)
 }
 
+func parseConsoleRemoteArrayPreview(logger *log.Logger, op *cdpruntime.ObjectPreview) string {
+	arr := make([]any, 0)
```

Comment:
Why specify len `0`? Shouldn't this be:
```suggestion
	arr := make([]any, 0, len(op.Properties))
```
To make the underlying array big enough for the properties we know in advance and avoid extra allocations?

Reply:
Resolved in 11be8b388289f450f9f52a2e065b5a59a993a496